config/routes/api.rb

```diff
@@ -150,6 +150,16 @@
       end
     end
 
+    namespace :notifications do
+      resources :requests, only: :index do
+        member do
+          post :accept
+        end
+      end
```

Comment:
There's a database column for whether the notification request is dismissed, but is there no API to actually dismiss a notification request?